src/app/deep-read/core/video_to_text/video_to_text_converter.py

```diff
@@ -0,0 +1,47 @@
+import os
+from .video_to_image_converter import Video_to_JPG_Converter
+from .image_to_text_converter import Image_to_Text_Converter
+from .text_to_sentence_converter import Text_to_Sentence_Converter
+from .screenclassifier import ScreenClassifier
+
+class Video_to_Text_Converter:
+
+    classifier = ScreenClassifier()
+
+    def __init__(self, video_path, offset_duration_seconds = 10):
+        self.video_path = video_path
+        self.render_path = os.path.splitext(self.video_path)[0] #rendered
+        self.offset_duration_seconds = offset_duration_seconds
+    
+    def extract_images_from_video(self):
+        video_to_jpg_converter = Video_to_JPG_Converter(self.video_path, self.render_path, self.offset_duration_seconds)
+        video_to_jpg_converter.convert()
+    
+    #Images are classified as screens or slides(presentation slides) using NN, we will convert only slide images into text
+    def get_slide_images_from_extracted_images(self):
+        self.slide_images = Video_to_Text_Converter.classifier.predict("/media/ashwin/Current/sem_9/open_source/deep-read/src/app/deep-read/core/temp/sample_transcript1")
```

Comment:
path is absolute here

Reply:
removed this bug in [e8f6d24](https://github.com/Acquil/deep-read/pull/32/commits/e8f6d24f2f6d31c30db0df37abe1e4916adab56e)